lib/commands/generate.ts

```diff
@@ -0,0 +1,106 @@
+import { utimes } from "fs-extra";
+import * as path from "path";
+import { TemplateManager } from "../TemplateManager";
+import { Util } from "../Util";
+import { default as config } from "./config";
+
+const command = {
+	command: "generate",
+	aliases: ["g"],
+	templateManager: null,
+	desc: "Generates new project",
+	builder: yargs => {
+		yargs
+			.command({
+				command: "template [name] [framework] [type] [skip-config]",
```

Comment:
This should stop at `"template [name]"` so the others can be normal flags (disregarding position)